src/consensus/aft/raft.h

```diff
@@ -1532,30 +1877,50 @@ namespace aft
       try_send_sig_ack({r.term, r.last_log_idx}, result);
     }
 
-    void recv_append_entries_signed_response(SignedAppendEntriesResponse r)
+    bool recv_append_entries_signed_response(
```

Comment:
What does the new boolean return value mean? Is this re-execution status or error?